server/index.js

```diff
@@ -119,10 +119,9 @@ export default class Server {
     await this.prepare()
     this.http = http.createServer(this.getRequestHandler())
     await new Promise((resolve, reject) => {
-      this.http.listen(port, (err) => {
-        if (err) return reject(err)
-        resolve()
-      })
+      this.http.on('error', reject)
+      this.http.on('listening', () => resolve())
+      this.http.listen(port, () => {})
```

Comment:
Seems it's optional https://nodejs.org/api/http.html#http_server_listen_handle_callback